grails-app/domain/org/pih/warehouse/requisition/Requisition.groovy

```diff
@@ -393,8 +393,8 @@ class Requisition implements Comparable<Requisition>, Serializable {
             requestedBy: requestedBy?.name,
             createdBy: createdBy?.name,
             updatedBy: updatedBy?.name,
-            dateCreated: dateCreated,
-            lastUpdated: lastUpdated,
+            dateCreated: dateCreated?.format("MMM dd, yyyy"),
+            lastUpdated: lastUpdated?.format("MMM dd, yyyy"),
```

Comment:
is this is a good idea to set date format on backend or should I have formatted the date on frontend ?

Reply:
sure it is